tomls/omnibus-base-mainnet-andromeda/perps/w-invokes.toml

```diff
@@ -0,0 +1,147 @@
+include = [
+    # TODO: this is a funny necessary dependency because the oracle manager is registering the same node twice
+    # however, it will not emit "NodeRegistered" the second time because its an unnecessary call
+    "../../oracles/pyth-w.toml",
+    "../../markets/common/bigcap-settings.toml",
+]
+# NOTE: This toml will live until the setMaxMarketValue is on mainnet
+
+[setting.perpsWMarketId]
+
+[setting.perpsWSkewScale]
+
+[setting.perpsWMaxFundingVelocity]
+
+[setting.perpsWMakerFeeRatio]
+
+[setting.perpsWTakerFeeRatio]
+
+[setting.perpsWMaxMarketSize]
+
+[setting.perpsWInitialMarginRatio]
+
+[setting.perpsWMaintenanceMarginFraction]
+
+[setting.perpsWFlagRewardRatioD18]
+
+[setting.perpsWMaxLiquidationLimitAccumulationMultiplier]
+
+[setting.perpsWMaxSecondsInLiquidationWindow]
+
+[setting.perpsWMaxLiquidationPd]
+
+[setting.perpsWEndorsedLiquidator]
+
+[setting.perpsWMinimumPositionMargin]
+
+[setting.perpsWLockedOiRatio]
+
+[invoke.createWPerpsMarket]
```

Comment:
When we add new perps markets we need to add price updates to e2e tests for all of them. This would be the reason for tests failing. See example in #171